.gitignore

```diff
@@ -1,2 +1,2 @@
-revengeMusic
-fmod/api/lowlevel/*
```

Comment:
If you're gonna remove this ignore then maybe the cmake files should search /usr/libs and /usr/include for the libs/header files.